modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -57,11 +57,7 @@
 
 	function dotnetbase.projectElement(prj)
 		if dotnetbase.isNewFormatProject(prj) then
-			if prj.flags.WPF then
-				_p('<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">')
-			else
-				_p('<Project Sdk="Microsoft.NET.Sdk">')
-			end
+			_p('<Project Sdk="%s">', dotnetbase.netcore.getsdk(prj))
```

Comment:
You need to change all instances of `flags.WPF` to check `dotnetsdk == "WindowsDesktop"`, otherwise when specifying `dotnetsdk "WindowsDesktop"` instead of `flags { "WPF" }` users won't get the expected output.